cli/src/cli_util.rs

```diff
@@ -2311,6 +2311,14 @@ pub fn parse_args(
     Ok((matches, args))
 }
 
+pub fn format_template<T>(ui: &Ui, arg: &T, template_language: &dyn Template<T>) -> String {
```

Comment:
nit: `template`, not `template_language`

Reply:
Done.